README.md

```diff
@@ -70,6 +70,10 @@ The `yarn build` step will download the latest TinyMCE package and generate new
 
 The TinyMCE API documentation is maintained within the [TinyMCE project repository](https://github.com/tinymce/tinymce) and compiled for the documentation site using [MoxieDoc](https://github.com/tinymce/moxiedoc).
 
+To update the published API documentation, change the version in the `.api-version` file, run `yarn build` and then commit the changes. Running `yarn build` will download the TinyMCE package specified in `.api-version` and generate new API reference content from source code.
```

Comment:
```suggestion
To update the published API documentation:

1. change the version in `.api-version`.
2. run `yarn build`.
3. commit the changes.

Running `yarn build` downloads the TinyMCE package specified in `.api-version` and generates new API reference content from source.
```
Putting procedures in lists makes them easier to follow (and makes folks slightly less likely to skip a step).

Also some minor edits.